README.md

```diff
@@ -184,6 +185,60 @@ const fileLastModifiedDate = preval.require('./get-last-modified-date', '../../s
 const fileLastModifiedDate = '2017-07-04'
 ```
 
+### preval file comment (`// @preval`)
```

Comment:
Could you add a note before the example explaining that while the previous uses change your import/require/assignment to the prevaled file, this one actually changes the whole file itself.